ministryofjustice/laa-apply-for-criminal-legal-aid
# CRIMAPP-649 Update NINO rule for evidence prompt

## Description of change

This PR updates Rule 32: National Insurance Number 

## Link to relevant ticket

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:

### After changes:

## How to manually test the feature

For the following scenarios, start an application for an applicant who is over 18 and do not enter a NINO.
1. Select case type 'Either way'. When you get to the relevant page, select one of the passporting benefits. Continue the application to the evidence upload page. You should see the text 'their National Insurance number' under the heading 'Other evidence of your client's income, outgoings or capital'. 
2. Select case type 'Either way'. Do not select any passporting benefit. When you get to the `income_benefits` page, select one or more benefits. Continue the application to the evidence upload page. You should see the text 'their National Insurance number' under the heading 'Other evidence of your client's income, outgoings or capital'.
3. Select case type 'Indictable'. Do not select any passporting benefit or non-passporting benefit. Continue the application to the evidence upload page. You should see the text 'their National Insurance number' under the relevant heading.
4. Select case type 'Already in crown court'. Do not select any passporting benefit or non-passporting benefit. Continue the application to the evidence upload page. You should see the text 'their National Insurance number' under the relevant heading.
5. Select case type 'Either way'. Do not select any passporting benefit or non-passporting benefit. Continue the application to the evidence upload page. You should **NOT** see the text 'their National Insurance number'. 

Scenario 6: Start an application for an applicant who is 18 or over. Enter a NINO. Do whatever you like for the rest of the application. Continue the application to the evidence upload page. You should **NOT** see the text 'their National Insurance number'.
